### DISCUSSION TITLE
管理依赖问题

### DISCUSSION THREAD

**Discussion by 525716612** [nonebot/discussions#370] [问答]
我现在有一个nonebot2的机器人，其中有的插件是直接下载的插件商店中插件源码，有时候插件更新的时候依赖会变更，这个时候我需要手动添加虚拟环境的依赖才能让这个插件跑起来，有没有什么比较方便能管理插件的依赖的方法

**Comment by yanyongyu** [comment]
[poetry](https://python-poetry.org/) [pdm](https://pdm.fming.dev/latest/)
